src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,9 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public static String getRandomColor() {
+        return String.valueOf(Colors.values()[new Random().nextInt(Colors.values().length)]);
```

Comment:
It's better practice to have only one Random instance for multiple calls. Consider creating a private final field Random instance.

Reply:
done